webapp/config.py

```diff
@@ -30,6 +31,14 @@ def __getattr__(self, name: str) -> Any:
         message = f"'{name}' not a valid configuration variable"
         raise AttributeError(message)
 
+    def _getenv(self, name: str) -> Any:
+        if value := os.getenv(name):
+            return value
+        if name in self.REQUIRED_ENV_VARS:
+            message = f"'{name}' is a required environment variable."
+            raise AttributeError(message)
+        return None
```

Comment:
This is used by the `@property` env access methods. This differs from `__getattr__`, which is only called for attributes that are not explicitly set for the class.

The purpose of `_getenv()` is to implement some error handling so that the `@property` env access methods can have a return type of `str`, which circumvents the following `mypy` error when [`ECSClient` is instantiated](https://github.com/MITLibraries/alma-sapinvoices-ui/pull/2/files#diff-b90418e2bed6dc92f2077fe7a7720859e6473682f422be94101d9480d229b04dR22-R29)